ui/src/components/TimePicker/TimePicker.vue

```diff
@@ -1,6 +1,6 @@
 <template>
   <div class="fd-time-picker">
-    <Popover noArrow :popoverVisible="false">
+    <Popover :uid="uid" noArrow :popoverVisible="false">
```

Comment:
@ChristianKienle Can I do this? The uid in popover is getting random, which cause the snapshot mismatch.

Reply:
Yes. I think this is fine.